hafnian/hafnian.pyx

```diff
@@ -361,3 +363,25 @@ def perm_real(double [:, :] A, quad=True, fsum=False):
 
     return permanent(mat)
 
+
+# ==============================================================================
+# Batch hafnian
+
+def hermite_multidimensional(double complex[:, :] A, double complex[:] d, int resolution, ren=False):
+    cdef int i, j, n = A.shape[0]
+    cdef vector[double complex] R_mat, y_mat
+	
+    cdef int renorm = 0
+
+    if ren:
+        renorm = 1
+
+    for i in range(n):
+        for j in range(n):
+            R_mat.push_back(A[i, j])
+
+
+    for i in range(n):
+        y_mat.push_back(d[i])
+
+    return hermite_multidimensional_cpp(R_mat, y_mat, resolution, renorm)

```

Comment:
I'm a bit confused; if this is the 'batch_hafnian', should we just call the function 'batch_hafnian' instead of hermite_multidimensional?

Reply:
No. As far as I know this connection has not been made and I think is better to keep them separated. The multidimensional hermite polynomials are a thing that has existed for centuries. The fact that they can be used to calculate batches of hafnians is a new finding...